### DISCUSSION TITLE
Version 1.26.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#4579] [Changelog]
## Bug fixes
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not desaturate the validation errors. (prio-3)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to the unsupported browser error. (prio-4)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to the alerts. (prio-4)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to studio invitations. (prio-3)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to mute messages and modals. (prio-4)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to remix credits. (prio-3)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to "comment globally disabled" message. (prio-4)
- [#4396](https://github.com/ScratchAddons/ScratchAddons/pull/4396) dark-www does not apply to the link in "What's Happening". (prio-4)
- [#4397](https://github.com/ScratchAddons/ScratchAddons/pull/4397) folders' preview generation is not optimized. (prio-3)
- [#4457](https://github.com/ScratchAddons/ScratchAddons/pull/4457) ctrl-enter-post does not accept Cmd key. (prio-3)
- [#4460](https://github.com/ScratchAddons/ScratchAddons/pull/4460) scratchr2 shows unusable scrollbars on profiles in Firefox. (prio-4)
- [#4483](https://github.com/ScratchAddons/ScratchAddons/pull/4483) scratchstats does not comma-separate "Most loves by country". (prio-4)
- [#4490](https://github.com/ScratchAddons/ScratchAddons/pull/4490) Easter egg addons are included in the addon count. (prio-4)
- [#4495](https://github.com/ScratchAddons/ScratchAddons/pull/4495) forum-search does not function in most cases. (prio-2)
- [#4495](https://github.com/ScratchAddons/ScratchAddons/pull/4495) forum-search is incompatible with scratchr2 addon. (prio-4)
- [#4499](https://github.com/ScratchAddons/ScratchAddons/pull/4499) scratchr2 does not style the green "alert" shown when following, etc. (prio-4)
- [#4500](https://github.com/ScratchAddons/ScratchAddons/pull/4500) animated-thumb displays the preview with wrong aspect ratio. (prio-3)
- [#4539](https://github.com/ScratchAddons/ScratchAddons/pull/4539) 60fps causes stage monitors to miss updates. (prio-3)
- [#4566](https://github.com/ScratchAddons/ScratchAddons/pull/4566) more-links linkifies existing links. (prio-4)
- [#4567](https://github.com/ScratchAddons/ScratchAddons/pull/4567) scratchstats graph renders incorrectly for some profiles. (prio-4)
- [#4575](https://github.com/ScratchAddons/ScratchAddons/pull/4575) dark-www affects the stage monitor on project page. (prio-3)
- [#4576](https://github.com/ScratchAddons/ScratchAddons/pull/4576) debugger causes a log spam when switching sprites. (prio-4)

## Features
- [#4267](https://github.com/ScratchAddons/ScratchAddons/pull/4267) New addon: default-costume-editor-color (prio-3)
- [#4324](https://github.com/ScratchAddons/ScratchAddons/pull/4324) New addon: sounds-duration (prio-3)
- [#4376](https://github.com/ScratchAddons/ScratchAddons/pull/4376) Add "View settings" button on the settings page. (prio-3)
- [#4531](https://github.com/ScratchAddons/ScratchAddons/pull/4531) Add selected category name to no results message. (prio-4)

**Comment by MaterArc** [comment]
Whoa! Cool update ^^
